project/Dependencies.scala

```diff
@@ -13,23 +13,30 @@ object Dependencies {
   lazy val scalaTestVersion = settingKey[String]("The version of ScalaTest to use.")
   lazy val scalaCheckVersion = settingKey[String]("The version of ScalaCheck to use.")
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
+  lazy val sslConfigVersion = settingKey[String]("The version of ssl-config to use.")
   val junitVersion = "4.12"
-  val sslConfigVersion = "0.3.7"
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
   val aeronVersion = "1.15.1"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.12.8", "2.13.0-M5"),
+    crossScalaVersions := Seq("2.12.8", "2.13.0-RC2"),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
     scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.14.0"),
-    scalaTestVersion := "3.0.7",
+    scalaTestVersion := "3.0.8-RC4",
```

Comment:
This answers my junit import question ;)

Reply:
Right: so there were some package changes between scalatest 3.0.7 and later versions. while there are (deprecated) forwarders in 3.0.8-RC4 that make it (mostly) source-compatible, I'm not convinced they are binary compatible.

Even though we only pull in scalatest in testkit projects, and we don't promise binary compatibility there, it might not be nice to force users (even 2.12 users) to update to an RC version of scalatest.

This might be a reason for postponing merging this PR.

Thinking about this we could switch this version based on scala version and use the source-compatible forwarders. That way it won't impact 2.12 users and will still be usable for RC2 users.